compiler/src/codegen/compcore.re

```diff
@@ -3540,6 +3569,7 @@ let compile_wasm_module = (~env=?, ~name=?, prog) => {
         Features.mutable_globals,
       ],
     );
+  let _ = Module.set_low_memory_unused(1);
```

Comment:
Is `1` the amount of memory unused?

Reply:
It's actually a number that represents a boolean. 😶

Binaryen v101 converts all these awkward number-as-boolean values to actual booleans, so that's something to look forward to.

Reply:
Good good! I thought that might be the case and was going to ask if we should do that massaging ourselves in binaryen.ml, but it seems it'll be solved for us!